### DISCUSSION TITLE
recipes/ingredients/sponge/angel-food-cake/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#971] [Comments]
# recipes/ingredients/sponge/angel-food-cake/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/sponge/angel-food-cake/

**Comment by nicholaswilde** [comment]
To avoid air pockets, spoon half the batter into the pan and level batter at a time with a small offset spatula.
